docs/user/installing-apps.md

```diff
@@ -1,23 +1,80 @@
-> **Caution**
->
-> The screenshots and information in this section is stale and in
-> process of being updated.
-
 ## Installing Apps into DHIS2 { #apps_installing_apps }
 
-Apps can be installed by uploading zip file into the App Manager. In,
-Services → Apps, click on the _App Hub_ menu item.
-![](resources/images/apps/app-management.png) The app can be uploaded by
-pressing the Browse button and after selecting the zip package, the file
-is uploaded automatically and installed in DHIS2. You can also browse
-through apps in the DHIS2 [AppHub](https://apps.dhis2.org/)
-and download apps from there. The DHIS2 AppHub allows for app
-searching, reviewing, commenting, requesting features, rating on the
-apps by the community.
-
-## Launching Apps { #apps_launching_apps }
-
-After installation, your apps will be integrated with the menu system
-and can be accessed under services and from the module overview page. It
-can also be accessed from the home page of the apps module. Click on an
-app in the list in order to launch it.
+Applications are one of the main ways that DHIS2 can be extended. A DHIS2 installation comes bundled with around 30 _core_ apps.
+
+These apps can be accessed from the search menu on the header bar - top image below - or the more powerful command palette since DHIS2 v42 (bottom image).
```

Comment:
```suggestion
These apps can be accessed from the search menu on the header bar (top image below) or the more powerful command palette since DHIS2 v42 (bottom image).
```